spec/index.bs

```diff
@@ -873,6 +927,103 @@ interface GPUBufferUsage {
 
 ## Buffer Mapping ## {#buffer-mapping}
 
+An application can request to map a {{GPUBuffer}} to get its mapping which is
+an {{ArrayBuffer}} representing the {{GPUBuffer}}'s allocation. Mappings are
+requested asynchronously so that the user agent can ensure the GPU is finished
```

Comment:
something here doesn't read well: "GPU is finished using ...", it reads like a passive tense here. Could we say "GPU is done" (which doesn't imply passive to me), or reverse the order and say "GPUBuffer is finished being used by the GPU" or something like that?

non-native speaker here, please feel free to discard if I'm just not reading this correctly :)